core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -128,6 +132,12 @@ object Boot extends App with LazyLogging {
       workbenchMetricBaseName = metricsPrefix
     )
 
+    val bigQueryDAO = new HttpGoogleBigQueryDAO(
+      gcsConfig.getString("appName"),
+      Json(FileUtils.readFileToString(new java.io.File(gcsConfig.getString("pathToBigQueryJson")), Charset.defaultCharset())),
+      metricsPrefix
+    )
```

Comment:
elsewhere in rawls.conf, we store the *contents* of a service account's json as a value - see `gcs.secrets` or `gcs.tokenSecretsJson`. If you followed that model - which would drive a change in firecloud-develop - you wouldn't have to read the file from the filesystem.